packages/core-transactions/src/handlers/htlc-claim.ts

```diff
@@ -175,9 +175,7 @@ export class HtlcClaimTransactionHandler extends TransactionHandler {
             amount: lockTransaction.amount,
             recipientId: lockTransaction.recipientId,
             timestamp: lockTransaction.timestamp,
-            vendorField: lockTransaction.vendorField
-                ? Buffer.from(lockTransaction.vendorField, "hex").toString("utf8")
```

Comment:
This needs to stay, only the HTLC lock is wrong. The reverts get Buffers from the database.